### ISSUE TITLE
Working group call #21

### ISSUE THREAD

**Issue by blockstackers** [stacksgov/pm#84]
## **Meeting Details**
**Date/Time:** Thursday, August 6, 2020 @ 4:00pm UTC / 12:00pm ET / [Time Zone Conversion](https://www.timeanddate.com/worldclock/fixedtime.html?msg=Stacks+Governance+Working+Group+Call&iso=20200806T16&p1=%3A&ah=1)
**Main Focus:** Grant proposal kickoff
**Location:** https://zoom.us/j/9468184266

**Agenda:**

- Discuss: What outstanding work do we still have to complete? Can any of that work be accommodated by the gov working group grant proposal?
- Recap of grant proposal goals
- Discuss potential deliverables/workstreams
- Start outline for grant

**Before next Wednesday's meeting, please do the following:**
- Add ideas to this [Grant Proposal working doc](https://paper.dropbox.com/doc/Gov-Group-Grant-Proposal--A402pAn8MQVQPIikglkCKxXXAQ-RN1efxJdptsRGVrJIbwou)
- Review Grants Application - Draft #85

**Comment by whoabuddy** [comment]
FYI - updated the agenda above to the correct date/time [per this post](https://discordapp.com/channels/621759717756370964/675389252816732166/738864790541303889) in Discord from Jenny and added link to GitHub issue for Grants Application #85.

**Comment by blockstackers** [comment]
Good catch @whoabuddy - thanks!

**Comment by whoabuddy** [comment]
Also, we got a bit out of order in regard to [our previous meetings](https://stacksgov.github.io/resources/#/calls/README?id=governance-working-group) and [open agenda issues](https://github.com/stacksgov/pm/issues?q=is%3Aissue+is%3Aopen+label%3Amtg-agenda), so I'm not sure the best way to proceed without community input.
- Call 19 happened and issue was closed #71 
- Call 20 did not happen, issue is still open with rolling set of topics from previous meetings #72 
- Retrospective call did happen in lieu of call 20, but issue is still open #82 

Do we want to continue tracking meetings the same way we have been, or is this prompting a change in format?

For example, we can call the retrospective call the 20th call and document it that way, but did we have a recording to post in addition to @ryanarndtcm's notes? What should we do with the agenda items in #72?
